src/mail.sh

```diff
@@ -761,8 +787,23 @@ function parse_mail_options()
         options_values['NO_INTERACTIVE']=1
         shift
         ;;
+      -[[:digit:]])
+        # if this is given without a commit reference we need to avoid adding a
+        # assumed commit
+        commit_count="$1"
+        commit_ref=1
+        shift
+        ;;
```

Comment:
Does this account for double-digits?

Reply:
Hi, no it does not. I think I added a note about it in the documentation, if not I will later. The problem is that the `get_opt` behavior doesn't allow for single dashes for options with more than one character. Also, the accepted options need to be given to `get_opt` and as far I could find there is no support for globs or patterns. Because of this I added a parser for these options when they are passed after the `--`.

Reply:
Please, correct me if I'm wrong, but we will not support things like `-10` or `-88`?

I wondered if we can handle this parameter outside the get_opt control, maybe check for this value in the "$@" and if we have it, we parse the info by ourselves and then update the "$@" array.